outputs.tf

```diff
@@ -2,7 +2,31 @@ output "ui" {
   sensitive = false
   value     = "http://${google_compute_instance.yugabyte_node.0.network_interface.0.access_config.0.nat_ip}:7000"
 }
+output "ssh_user" {
+  sensitive = false
+  value = "${var.ssh_user}"
+}
 output "ssh_key" {
   sensitive = false
   value     = "${var.ssh_private_key}"
-}
\ No newline at end of file
+}
+
+output "JDBC" {
+  sensitive =false
+  value     = "postgresql://postgres@${google_compute_instance.yugabyte_node.0.network_interface.0.access_config.0.nat_ip}:5433"
+}
+
+output "YSQL"{
+  sensitive = false
+  value     = "psql -U postgres -h ${google_compute_instance.yugabyte_node.0.network_interface.0.access_config.0.nat_ip} -p 5433"
```

Comment:
I'd recommend using `yugabyte` user instead of postgres.

Reply:
I will make those changes as a separate PR. Closing this PR as the changes are already in master.